src/main/java/edu/umich/ctools/sectionsUtilityTool/SectionsUtilityToolServlet.java

```diff
@@ -338,12 +325,133 @@ public void storeContext(Context context, HttpServletRequest request) {
 		// Fill context with the required lti values.
 		// The VelocityViewServlet will take care of sending the processing on
 		// to the proper velocity template.
-		fillCcmValuesForContext(ltiValues, request);
+		fillCcmValuesForContext(ltiValues, request,tc);
 
 		context.put("ltiValues", ltiValues);
 	}
 
-	private void fillCcmValuesForContext(Map<String, String> ltiValues, HttpServletRequest request) {
+	private boolean isAccountAdmin(HttpServletRequest request) {
+		M_log.debug("isAdmin() call");
+		String courseId = request.getParameter(CUSTOM_CANVAS_COURSE_ID);
+		int userId = Integer.parseInt(request.getParameter(CUSTOM_CANVAS_USER_ID));
+
+		int courseAccountId= getCourseAccountId(courseId);
+		if(courseAccountId == 0){
+			return false;
+		}
+		M_log.debug("************** " + courseAccountId);
+		return determineIfUserIsAdminInHierarchy(userId,courseAccountId);
+	}
+
+	private boolean determineIfUserIsAdminInHierarchy(int userCanvasId, int courseAccountId){
+		M_log.debug("determineIfUserIsAdminInHierarchy() call");
+		boolean isAdmin = false;
+		String url = canvasURL + "/api/v1/accounts/" + courseAccountId + "/admins/?user_id[]=" + userCanvasId;
```

Comment:
Maybe the 'canvasURL + "/api/v1/accounts/"' part could be replaced by a little method that takes the rest of the url as an argument?  IMHO that's easier to read and means there isn't much to change when canvas changes to api v2 or v3...

E.g. 
makeCanvasUrl(courseAccountId + "/admins/?user_id[]=" + userCanvasId);